keyboards/keyten/kt60hs_t/v1/keyboard.json

```diff
@@ -1,34 +1,22 @@
 {
-    "manufacturer": "keyten",
-    "keyboard_name": "kt60HS-T",
-    "maintainer": "key10iq",
     "bootloader": "stm32-dfu",
     "diode_direction": "COL2ROW",
-    "features": {
-        "bootmagic": true,
-        "command": false,
-        "console": false,
-        "extrakey": true,
-        "mousekey": true,
-        "nkro": true
-    },
     "matrix_pins": {
         "cols": ["A3", "A4", "A5", "A6", "A7", "B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15", "A14"],
         "rows": ["B15", "B14", "B12", "B13", "B0"]
     },
     "processor": "STM32F401",
     "usb": {
         "device_version": "0.0.1",
-        "pid": "0x6004",
-        "vid": "0xEB69"
+        "pid": "0x6004"
     },
     "community_layouts": [
         "60_ansi_wkl_split_bs_rshift", 
         "60_hhkb", 
         "60_ansi_tsangan_split_bs_rshift"
     ],
     "layout_aliases": {
-        "LAYOUT_60_tsangan_hhkb": "LAYOUT_60_ansi_tsangan_split_bs_rshift"
+        "LAYOUT_all": "LAYOUT_60_ansi_tsangan_split_bs_rshift"
```

Comment:
Prior layout name required for backwards/legacy compatibility
```suggestion
        "LAYOUT_60_tsangan_hhkb": "LAYOUT_60_ansi_tsangan_split_bs_rshift",
        "LAYOUT_all": "LAYOUT_60_ansi_tsangan_split_bs_rshift"
```

Reply:
Hmm... In my other PR you suggested me to do the opposite 

![image](https://github.com/user-attachments/assets/481b06df-e9cb-4e0f-9b22-50faadc4b9b0)